README.md

```diff
@@ -18,7 +18,7 @@
 {
   "parser": "babel-eslint",
   "extends": [
-    "@jetbrains",
+    "@jetbrains/eslint-config",
```

Comment:
Why the old way doesn't work anymore?

Reply:
We use old way at least in Hub, I'd like to find a way to keep it unchanged

Reply:
As i understand `@jetbrains` is just a scope name, not package, so does this actually works? If yes I will just remove this commit from PR.

Reply:
@almsh yes it works, it is feature of eslint shareable config
https://eslint.org/docs/developer-guide/shareable-configs#npm-scoped-modules

Reply:
eslint resolves `foo` as `eslint-plugin-foo` and `@foo` as `@foo/eslint-plugin`

Reply:
Thanks for clarification:)